modules/core/src/test/scala/iotatests/TypeEqv.scala

```diff
@@ -0,0 +1,58 @@
+package iotatests
+
+import cats._
+import org.scalacheck.Prop
+import org.scalacheck.Prop._
+import shapeless.{ Id => _, _ }
+import shapeless.ops.hlist.{ ToList => HListToList }
+
+import scala.reflect.runtime.universe._
+
+sealed trait TypeEqv[A] {
```

Comment:
We should use 'should compile' or some of the shapeless facilities to test for compilation.